api/src/main/java/org/pmiops/workbench/elasticsearch/ElasticSearchService.java

```diff
@@ -0,0 +1,55 @@
+package org.pmiops.workbench.elasticsearch;
+
+import org.elasticsearch.client.RequestOptions;
+import org.elasticsearch.client.RestHighLevelClient;
+import org.elasticsearch.client.core.CountRequest;
+import org.elasticsearch.index.query.AbstractQueryBuilder;
+import org.elasticsearch.index.query.BoolQueryBuilder;
+import org.elasticsearch.index.query.TermQueryBuilder;
+import org.elasticsearch.search.builder.SearchSourceBuilder;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.io.IOException;
+
+@Service
+public class ElasticSearchService {
+
+  private static final String INDEX = "index";
+  private static final String TYPE = "type";
+  private RestHighLevelClient client;
+
+  @Autowired
+  public ElasticSearchService(RestHighLevelClient client) {
+    this.client = client;
+  }
+
+  /**
+   * This method will be used to generate counts from a
+   * org.pmiops.workbench.model.SearchRequest.
+   *
+   * @param userRequest
+   * @return
+   */
+  public Long elasticCount(org.pmiops.workbench.model.SearchRequest userRequest) throws IOException {
```

Comment:
FYI: For this PR - totally fine to just have a placeholder proof-of-concept method here, e.g. returning cluster status.

Reply:
Will do. I'm running into some issues that need to be solved before i can do this: https://docs.google.com/document/d/1hwXHlOW7wBpAQrj0UygMnQnO1pfBiSVSsKsf6C6DirM/edit#